.github/workflows/docker.yml

```diff
@@ -0,0 +1,31 @@
+name: Docker
+
+on:
+  push:
+    branches: [ marine ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: "Cache docker build"
+      uses: actions/cache@v2
+      id: cache-docker
+      with:
+        path: '/var/lib/docker/'
+        key: 'constant-because-docker-handles-cache'
+
+    - name: "Build docker image"
+      run: docker build -t marine-core .
+
+
+    - uses: docker/build-push-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        repository: domerd/marine-core
```

Comment:
Add a TODO or something to change that from `domerd/` to something better when we can.
(Something better = `tomlegkov/` or so slash at all and just `marine-core`)